lib/vim_ahk.ahk

```diff
@@ -146,6 +152,7 @@ class VimAhk{
 
   SetExistValue(){
     for k, v in this.Conf {
+      ; Ideally we would be checking if the var exists, but that's tricky to do in AHK.
```

Comment:
Do not put such a comment in the source code.
If you have a suggestion, could you please give as an issue.

Reply:
It is specifically for people wondering why the function is being called, rather than a suggestion. I can re-word it if you were confused. It isn't a suggestion because checking if variables exist is too tricky to do in AHK, but effectively what we are doing in this function is ensuring variables _do_ exist.

Reply:
Yes, here we implemented the function that has an actually "tricky" way to check if the var exists or not.
And the function name is `SetExistValue`, so it is obvious what this function is doing.

The comment seems like that the function can not check the var existence and is doing something else.
But it is not true.

Reply:
`SetExistValue` doesn't make semantic sense in English, so no, it isn't obvious what it is doing. And this function doesn't check the existence at all, because a variable that exists with the value `" "` would still be changed. It is a workaround for not being able to check its existence.

Reply:
I have re-worded the comment. I think it is valuable info for future maintainers to read, because the function name doesn't actually make sense.